jsonrpsee/Cargo.toml

```diff
@@ -20,6 +20,6 @@ types = { path = "../types", version = "0.2.0", package = "jsonrpsee-types", opt
 
 [features]
 client = ["http-client", "ws-client"]
```

Comment:
perhaps we should add types here too? however types are re-exported from the clients